Themes/default/languages/index.english.php

```diff
@@ -475,6 +475,7 @@
 $txt['not_removed_extra'] = '%1$s is a backup of %2$s that was not generated by SMF. It can be accessed directly and used to gain unauthorized access to your forum. You should delete it immediately.';
 $txt['generic_warning'] = 'Warning';
 $txt['agreement_missing'] = 'You are requiring new users to accept a registration agreement, however, the file (agreement.txt) does not exist.';
+$txt['policy_agreement_missing'] = 'You are requiring new users to accept a privacy policy, however, the privacy policy is empty.';
```

Comment:
```suggestion
$txt['policy_agreement_missing'] = 'You are requiring new users to accept a privacy policy; however, the privacy policy is empty.';
```

Reply:
Should I change the agreement_missing above as well?